homeassistant/components/mqtt/siren.py

```diff
@@ -275,15 +292,15 @@ def state_message_received(msg):
             if json_payload:
                 # process attributes
                 try:
-                    vol.All(TURN_ON_SCHEMA)(json_payload)
+                    params = vol.All(TURN_ON_SCHEMA)(json_payload)
                 except vol.MultipleInvalid as invalid_siren_parameters:
                     _LOGGER.warning(
                         "Unable to update siren state attributes from payload '%s': %s",
                         json_payload,
                         invalid_siren_parameters,
                     )
                     return
-            self._update(process_turn_on_params(self, json_payload))
+                self._update(process_turn_on_params(self, params))
```

Comment:
This looks like a bug fix :)
Should the next line also be indented?

Reply:
> This looks like a bug fix :)

Yeah, we should pass the validated payload, not the raw data.

> Should the next line also be indented?

No because the payload van be a state const or a json payload and we do want to write the state in both cases.